pkg/nuget/parse_testcase.go

```diff
@@ -0,0 +1,59 @@
+package nuget
```

Comment:
I suggest moving these vars over to `pkg/nuget/parse_test.go` so it's easier to read while looking at the test.

Reply:
Hmm - I see why you did this as the rest of project follows it. I take back what I said in that case :) My preference would still be to keep it together but I guess it doesn't hurt otherwise.

Reply:
I felt the same when I wrote it, but yeah, wanted to keep the code as similar as possible to the other source :)

Reply:
That is because test cases are automatically generated by commands as follows because it is impossible to create the cases manually.
https://github.com/aquasecurity/go-dep-parser/blob/master/pkg/yarn/parse_testcase.go

It is too big and if we put it into a test file, it becomes difficult to distinguish between test cases and test logic. Also, the auto-generated file should be separated from the normal file. In such a case, it's common to define test cases in the dedicated file.

In this case, test cases are small enough. If we can maintain it manually, we can merge it into `parse_test.go`.